deploy/csi-operator.yaml

```diff
@@ -89,10 +89,64 @@ spec:
   group: snapshot.storage.k8s.io
   names:
     kind: VolumeSnapshotClass
+    listKind: VolumeSnapshotClassList
     plural: volumesnapshotclasses
+    singular: volumesnapshotclass
   scope: Cluster
-  version: v1alpha1
-
+  preserveUnknownFields: false
```

Comment:
comment it out, and put note saying that suported in 1.15+

Reply:
done